lib/const.rb

```diff
@@ -1,6 +1,6 @@
 # Defines common constants used in different parts of crew
 
-CREW_VERSION = '1.20.3'
+CREW_VERSION = '1.20.5'
```

Comment:
Did you merge master into your working branch?  I'm thinking the old line should be `CREW_VERSION = '1.20.4'`.

Reply:
The version is bumped

Reply:
Merged just now (seems I misunderstood your meaning, never mind😅)